code/__DEFINES/genetics.dm

```diff
@@ -146,4 +146,5 @@
 #define RESISTHOT		16
 #define RESISTCOLD		17
 #define NO_EXAMINE		18
-#define CAN_WINGDINGS	19
\ No newline at end of file
+#define CAN_WINGDINGS	19
+#define NO_BONES        20
```

Comment:
We're trying to get away from having flags determine abilities, where possible; it'd be better if we did what we did with Diona, where they have a limb subtype, and that subtype has the `unbreakable` var set to `TRUE`.

Reply:
Would the var reset if the limb was removed? I can see people amputating their boneless limbs and giving them to others